README.md

```diff
@@ -72,7 +72,14 @@ Download the latest binary executable for your operating system:
   dnf install tektoncd-cli
   ```
 
-If you have [go](https://golang.org/) installed, `GO111MODULE="on" go get github.com/tektoncd/cli@v0.5.0` is all you need!
+* Source install, If you have [go](https://golang.org/) installed and you want
```

Comment:
Thanks will amend, 

fyi github has a nifty "Insert a suggestions" button, so this can go quicker :) 
 
![image](https://user-images.githubusercontent.com/98980/68867431-15760900-06f6-11ea-857a-f7865d6ac8f9.png)

Reply:
Oh nice! I've seen that used but haven't looked into how yet.